ui/src/app/machines/components/ActionFormWrapper/CloneForm/CloneResults/CloneResults.tsx

```diff
@@ -171,6 +175,9 @@ export const CloneResults = ({
         },
         []
       ).length;
+  const viewingMachineDetails = pathname.startsWith(
```

Comment:
I'm always a little worried that using the URL will result in a bug later on. What do you think about passing a prop down from MachineHeader?

Reply:
There’s a test that should fail if we ever change the machine details URL, but I think you’re right that it’s a bit brittle. I’ll add a ‘viewingDetails’ prop or something